lib/sdr_client/deposit.rb

```diff
@@ -14,7 +14,7 @@ def self.run(label: nil,
                  viewing_direction: nil,
                  access: 'dark',
                  download: 'none',
-                 use_statement: nil,
+                 use_and_reproduction: nil,
```

Comment:
does license need to go here and/or below?

Reply:
It might make sense at some point to add it but changing deposit abilities was not part of #186. (Only changing update abilities.)